pkg/cmd/show.go

```diff
@@ -375,7 +379,7 @@ func showKibana(targetReader TargetReader) {
 	if len(target.Target) == 2 {
 		namespace = "garden"
 	} else if len(target.Target) == 3 {
-		namespace = getTechnicalID()
+		namespace = getFromTargetInfo("shootTechnicalID")
```

Comment:
nothing wrong with this line, however we should add an issue to replace checks like len(target.Target) == 3 with more descriptive getters
```go
if len(target.Target) == 2 {
  namespace = "garden"
} else if len(target.Target) == 3 {
  namespace = getFromTargetInfo("shootTechnicalID")
}
```

instead, something like 
```go
if !isShootTargeted() {
  namespace = "garden"
} else {
  namespace = getFromTargetInfo("shootTechnicalID")
}
```
makes the code more readable

Reply:
done

Reply:
there are a lot of other places with this pattern, please create a separate issue so that it will be cleaned up as you already did for these few lines

Reply:
sure. https://github.com/gardener/gardenctl/issues/347

Reply:
yes https://github.com/gardener/gardenctl/issues/347